completions/docker.fish

```diff
@@ -22,504 +22,504 @@ function _halostatue_fish_docker_no_subcommand --description 'Test if docker has
 end
 
 # common options
-complete --command docker --description "Location of client config files (default \"/Users/austin/.docker\")" --condition '_halostatue_fish_docker_no_subcommand' --required --long-option config
-complete --command docker --description "Name of the context to use to connect to the daemon (overrides" --condition '_halostatue_fish_docker_no_subcommand' --exclusive --short-option c --long-option context
+complete --command docker --description "Location of client config files (default \"/Users/jroeder/.docker\")" --condition '_halostatue_fish_docker_no_subcommand' --require-parameter --long-option config
```

Comment:
Mmm. I think we may want to add a specific override for this.

Reply:
Good catch - I didn't even notice my username in there!

Reply:
resolved in 48ed981